### ISSUE TITLE
Method `havingCount` results in error

### ISSUE THREAD

**Issue by mookman288** [ParticleBits/pdo#1]
Error:
```
SQLSTATE[42S22]: Column not found: 1054 Unknown column 'COUNT( * )' in 'having clause'
```

The same is true of `having('COUNT(*)', ...`

The workaround is to assign COUNT(*) AS count, and then reference the count variable in the SQL. 

I know we discussed this previously, but this month has been unexpectedly busy. I will see if I can figure out where this problem occurs and a good method to resolve it if I have time.

**Comment by mookman288** [comment]
Could this be related to the backticks? 

```
$this->container[] = " $chainType `$column` $operator ?";
```

**Comment by mikegioia** [comment]
@mookman288 It's definitely related to the backticks. I went through and I think this could be addressed by using `havingCount()`? That function also takes a column name but adds in the `COUNT` for you.

**Comment by mookman288** [comment]
I was using havingCount, the method, originally. It still uses the backticks. Are they necessary (they aren't in the original library) or were they a design feature?

**Comment by mikegioia** [comment]
Oh I understand now. I'll send a PR up in a few minutes with my proposed changes.

**Comment by mikegioia** [comment]
@mookman288 I know this is a trivial update but would you mind just testing the branch over at #2 to see if it resolves it. I know it will because it's what you originally suggested and I see the problem. I didn't realize `having()` was called by the other methods in that class until now.

**Comment by mookman288** [comment]
I just tested this (manually, I didn't pull anything down) and it works.

**Comment by mikegioia** [comment]
Ok this should be resolved now in master at version 2.0.3!
